frontend/src/component/playground/Playground/PlaygroundResultsTable/FeatureResultInfoPopoverCell/FeatureStrategyList/StrategyList/StrategyItem/StrategyExecution/ConstraintExecution/ConstraintExecution.tsx

```diff
@@ -11,6 +15,7 @@ import { ConstraintOk } from './ConstraintOk/ConstraintOk';
 interface IConstraintExecutionProps {
     constraints?: PlaygroundConstraintSchema[];
     input?: PlaygroundRequestSchema;
+    showExecutionResult?: boolean;
```

Comment:
boolean props are usually a component design smell. Can we create 2 separate single responsibility components: ConstraintsWithResult and ConstraintWithoutResults?

Reply:
components are functions and functions in general should avoid boolean parameters: https://softwareengineering.stackexchange.com/questions/147977/is-it-wrong-to-use-a-boolean-parameter-to-determine-behavior

Reply:
makes sense